pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/PersistentTopicStreamingDispatcherE2ETest.java

```diff
@@ -25,7 +25,7 @@
 /**
  * PersistentTopicE2ETest with {@link StreamingDispatcher}
  */
-@Test(groups = "flaky")
+@Test(groups = "broker")
```

Comment:
The stream-api implementation should be unstable, and I think the class should also be marked `flaky`

Reply:
done.